app/ui/src/main/java/com/fsck/k9/ui/messagelist/MessageListAppearance.kt

```diff
@@ -0,0 +1,23 @@
+package com.fsck.k9.ui.messagelist
+
+import com.fsck.k9.K9
+
+data class MessageListAppearance(
```

Comment:
I'd rather have this be a simple data class without a dependency to `K9`. Please pass the respective values when creating the instance instead.

Reply:
done